package.json

```diff
@@ -18,17 +18,18 @@
     "sanity"
   ],
   "dependencies": {
-    "@sanity/base": "^2.13.0",
-    "@sanity/components": "^2.13.0",
-    "@sanity/core": "^2.13.0",
-    "@sanity/default-layout": "^2.13.0",
-    "@sanity/default-login": "^2.13.0",
-    "@sanity/desk-tool": "^2.13.0",
-    "@sanity/vision": "^2.13.0",
+    "@sanity/base": "^2.24.1",
+    "@sanity/components": "^2.14.0",
+    "@sanity/core": "^2.24.0",
+    "@sanity/default-layout": "^2.24.1",
+    "@sanity/default-login": "^2.24.1",
+    "@sanity/desk-tool": "^2.24.1",
+    "@sanity/vision": "^2.24.1",
     "prop-types": "^15.7",
     "react": "^17.0",
     "react-dom": "^17.0",
-    "sanity-plugin-intl-input": "^5.2.1"
+    "sanity-plugin-intl-input": "^5.2.1",
+    "styled-components": "^5.2.0"
```

Comment:
why adding this?

Reply:
@blushi these were actually installed automatically through running `sanity upgrade` - presumably they're used in sanity studio